spyder/plugins/editor/widgets/editor.py

```diff
@@ -2781,9 +2781,7 @@ def set_layout_settings(self, settings, dont_goto=None):
 
 
 class EditorWidget(QSplitter):
-    def __init__(self, parent, plugin, menu_actions, show_fullpath,
-                 show_all_files, group_cells, show_comments,
-                 sort_files_alphabetically):
+    def __init__(self, parent, plugin, menu_actions, oe_options):

```

Comment:
I suggest we rename this to something more self explanatory

For example: (I am open to suggestions) 
`oe_options`  -> `outline_options`